### DISCUSSION TITLE
05.Linux/31.Ubuntu%E5%AE%89%E8%A3%85%E4%BC%81%E4%B8%9A%E5%BE%AE%E4%BF%A1

### DISCUSSION THREAD

**Discussion by giscus** [mirrormingzZ/mirrormingzZ.github.io#31] [Show and tell]
# 05.Linux/31.Ubuntu%E5%AE%89%E8%A3%85%E4%BC%81%E4%B8%9A%E5%BE%AE%E4%BF%A1

A VitePress site

https://doc.pursue.pub/05.Linux/31.Ubuntu%E5%AE%89%E8%A3%85%E4%BC%81%E4%B8%9A%E5%BE%AE%E4%BF%A1.html

**Comment by ww-tj** [comment]
谢谢老哥

**Comment by magisk317** [comment]
不使用deepin版会怎么样
